CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
+* Starting Kubernetes 1.29, the default cgroups implementation on Azure Linux AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance, and read the [FAQ](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/aks-increased-memory-usage-cgroup-v2) for cgroupsv2.
+* Staring with the `2024-01-01` and `2024-01-02-preview` APIs, we will begin to reject unknown fields in the request payloads.
+* Changes to reduce the kube-reserved memory reservation and eviction threshold will not be available in 1.28 as previously shared due to a release issue. These optimizations will be releasing with AKS Kubernetes minor version 1.29, which previews in January 2024.
+ 
+### Release notes
+
+* Bug Fixes 
+  * Fix for artifact streaming update related panic when nil and switches testing to use params/test package functions.
+  * Fix for LTS to look at AgentPoolResources.
+  * Fix for a put MC operation with an API version that doesn't have the servicemeshprofile triggers an "invalid mode" error during RP validations.
+  * Fix for the wrong MCR URL for Keda in AGC caused by cloud Environment value not being passed to the addon chart.
+
+* Behavioral Change
+  * ManagedCluster, AgentPool are now deprecated fields in agentpool goals. As goal should be including all necessary information and is generated from ManagedCluster abnd AgentPool. Thus, they should no longer be used.
+  * The default ResponseHeaderTimeout (which specifies the amount of time to wait for a server's response headers after fully writing the request, does not include the time to read the response body) is now set to 55s to prevent abnormal slow operation caused by hanging connection.
+  * The memory limit for [Azure Key Vault provider for Secrets Store CSI Driver](https://learn.microsoft.com/azure/aks/csi-secrets-store-driver) is now increased to 300Mi.
+  * Expanders flag is removed from AutoscalerProfile from November API
+  * Config checksum is added to cns daemonset spec will roll CNS pods that need to pick up new config values in azure-cns-configmap, this is to resolve the recent Windows goroutine leak.
+    
+* Component Updates
+  * Windows Kubernetes CVE fixes for [CVE-2023-5528](https://github.com/advisories/GHSA-hq6q-c2x6-hmch)
+  * Update ama-logs addon image to [3.1.16](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md)
+  * Azure Windows 2019 Image has been updated to [Azure Windows - 17763.5206.231213](https://github.com/Azure/AKS/blob/2024-01-08/vhd-notes/AKSWindows/2019/17763.5206.231213.txt)
```

Comment:
Why is AZ linux (mariner) VHD missing here ?https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSCBLMarinerV2/gen2/202401.03.0.txt

Reply:
Thought we should be watching for AzureLinux, added and updated